config/config.go

```diff
@@ -21,6 +24,15 @@ type configDefinition struct {
 	ScanRules         []scanRule `koanf:"scan_rules"`
 }
 
+func (configDefinition configDefinition) GetWebhookInterval() time.Duration {
+	// not currently configurable.
+	return time.Second
```

Comment:
...and I'm not sure it should be.  I chose a second because small frequent deliveries means less for golbat to hold, and less for receiver to process in each drop.
Since it's all in memory this does not impact the database.
If anything, I'd make it smaller not bigger :-)

Reply:
Yes, I agree with all of that. :)